homeassistant/components/shopping_list/config_flow.py

```diff
@@ -0,0 +1,32 @@
+"""Config flow to configure ShoppingList component."""
+from homeassistant import config_entries
+
+from .const import DOMAIN  # pylint: disable=unused-import
```

Comment:
After `await self.async_set_unique_id(self._username)` disable pylint shouldn't be needed